static/app/components/onboardingPanel.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import emptyStateImg from 'sentry-images/spot/feedback-empty-state.svg';
+
+import {Button} from 'sentry/components/button';
+import ButtonBar from 'sentry/components/buttonBar';
+import OnboardingPanel from 'sentry/components/onboardingPanel';
+import JSXNode from 'sentry/components/stories/jsxNode';
+import JSXProperty from 'sentry/components/stories/jsxProperty';
+import storyBook from 'sentry/stories/storyBook';
+
+export default storyBook(OnboardingPanel, story => {
+  story('Default', () => {
+    return (
+      <Fragment>
+        <p>
+          The <JSXNode name="OnboardingPanel" /> component is used for creating an empty
+          state layout or onboarding display. It can be used to show an empty state image
+          side-by-side with onboarding content.
+        </p>
+        <p>
+          An example <JSXNode name="OnboardingPanel" /> with an image looks like this:
+        </p>
+
+        <OnboardingPanel image={<img src={emptyStateImg} />}>
```

Comment:
Would the default render be without an image, because image is optional?

Reply:
yup, that's why i said "example" and not "default" -- but i need to update the title of this section